cli/src/semgrep/commands/ci.py

```diff
@@ -296,18 +297,19 @@ def ci(
                     if state.env.semgrep_url != "https://semgrep.dev"
                     else ""
                 )
-                connection_task = progress_bar.add_task(
-                    f"Fetching configuration from Semgrep Cloud Platform{at_url_maybe}"
-                )
-                scan_handler.fetch_and_init_scan_config(metadata_dict)
-                progress_bar.update(connection_task, completed=100)
 
                 start_scan_task = progress_bar.add_task(
                     f"Reporting start of scan for [bold]{scan_handler.deployment_name}[/bold]"
```

Comment:
Many of the code changes in this PR were to enable us to know and report the deployment name back to the user. Previously, we only kept track of the deployment_name _after_ `fetch_and_init_scan_config` had been called